WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/MostPopularInsightsUseCase.kt

```diff
@@ -83,9 +89,19 @@ class MostPopularInsightsUseCase
                         )
                 )
         )
+        addActionCards()
         return items
     }
 
+    private fun addActionCards() {
+        val model = mostPopularStore.getMostPopularInsights(statsSiteProvider.siteModel)
```

Comment:
np: Can we pass domainModel to `addActionCards()` we already had instead of calling `getMostPopularInsights()` again? `getMostPopularInsights` will cause an extra SQL call.